mappings/net/minecraft/world/chunk/storage/RegionFileCache.mapping

```diff
@@ -1,2 +1,17 @@
 CLASS bta net/minecraft/world/chunk/storage/RegionFileCache
+	FIELD d regionFiles Lit/unimi/dsi/fastutil/longs/Long2ObjectLinkedOpenHashMap;
 	METHOD a getRegionFile (Lbby;)Lbsz;
+		ARG 2 regionId
+		ARG 4 regionFile
+		ARG 5 regionFolder
+		ARG 6 mcaFile
+		ARG 7 regionFile2
```

Comment:
What is this parameter?

Reply:
This is a local variable for a new RegionFile if the first one doesn't exist.
That's due to Mojang making everything final.